ContactManger/src/main/java/Manager.java

```diff
@@ -0,0 +1,45 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Manager {
+    private List<Contact> myFriends;
+    int friendsCount;
+
+    public Manager() {
+        friendsCount = 0;
+        myFriends = new ArrayList<>();
+    }
+
+    public void addContact(Contact contact) {
+        myFriends.add(contact);
+        friendsCount++;
+    }
+
+    public List<Contact> searchContact(String name) {
+        List<Contact> searchResult = new ArrayList<>();
+        for (Contact customer : myFriends) {
+            if (customer.getName().equals(name)) {
+                searchResult.add(customer);
+            }
+        }
+        return searchResult;
+    }
+
+    public void showContacts() {
+        int numerator = 0;
+        for (Contact contact : myFriends) {
+            System.out.println(numerator + 1 + "\t" + contact.getName() + "," + contact.getPhone()
+                    + "," + contact.getEmail());
+            numerator++;
+        }
+    }
+
+    public void removedContact(String name) {
+        for (Contact customer : myFriends) {
```

Comment:
@anamariaTapia Following your counter logix, you have here an issue, the counter of contacts should reduce, for this reason is better to handle the counter just as the contact list size